### DISCUSSION TITLE
Django

### DISCUSSION THREAD

**Discussion by duebbert** [codatio/codat-docs#575] [Q&A] [answered]
Hi,

We're starting to implement Codat to work with our Django application. To have to avoid reinventing the wheel, are there any examples out there where this has been done before?

Thanks

**Comment by dcoplowe** [comment, marked as answer]
Hi duebbert, 

Our python client libraries should work with django, however, we don't have any example projects today. I would recommend looking at our [python repo](https://github.com/codatio/client-sdk-python/tree/main/accounting) as it contains code snippets for every method available. For example, here is our documentation on how to access [accounts](https://github.com/codatio/client-sdk-python/blob/main/accounting/docs/sdks/accounts/README.md) from an accounting package. 

I hope that helps.

Many thanks,

David
